releases.md

```diff
@@ -29,7 +31,31 @@ For more information on the changes planned for each release, please see the [Pr
 | 1.3.6          | September 30, 2022 | October 6, 2022    |
 | 2.4            | November 3, 2022   | November 10, 2022  |
 | 1.3.7          | December 1, 2022   | December 8, 2022   |
-| 2.5            | January 10, 2023   | January 12, 2023   |
+| 2.5            | January 10, 2023   | January 17, 2023   |
+
+### Proposed 2023 schedule ###
+
+
+
+Note:  We have not added a major release to the 2023 schedule yet.  If/when we add one, it will replace a minor release in the 2.x line.  See below for criteria for a major releases.
+
+| Release Number| Code Freeze Date  | Release Date          |
+|:--------------|:------------------|:-------------------   |
+| 2.5.0         | January 10th      | January 17th          |
+| 1.3.8         | January 26th      | February 2nd          |
+| 2.6.0         | February 21th     | February 28th         |
+| 1.3.9         | March 9th         | March 16th            |
+| 2.7.0         | April 11th        | April 18th            |
+| 1.3.10        | May 11th          | May 18th              |
+| 2.8.0         | May 30th          | June 6th              |
+| 1.3.11        | June 22nd         | June 29th             |
+| 2.9.0         | July 11th         | July 18th             |
+| 1.3.12        | August 3rd        | August 10th           |
+| 2.10.0        | August 22nd       | August 29th           |     
+| 1.3.13        | September 14th    | September 21st        |
+| 2.11.0        | October 10th      | October 17th          |
+| 1.3.14        | November 9th      | November 16th         |
```

Comment:
We have gotten much better at preparing and releasing patches, what do you think about betting that we can compress the 7 day time to something smaller?  Being bold, what do you think about targeting releasing at the end of the day of code freeze?

Reply:
I asked @bbarani about this, and he asked to keep the 7 days.  are you really, really sure @bbarani we need to freeze for 7 days?